content/games/factions/routines.md

```diff
@@ -0,0 +1,143 @@
+Groups.
+
+When activating individual Groups, unless specified, they follow this process:
+
+1. Must activate the Captain's Group first, then the closest Group to the Captain's Group which has
+   not yet activated.
+1. Roll 1d6 to determine the activation: Move (1--2), Attack (3--4), Shoot (5--6). If the group
+   cannot perform the specified action, default to the next lower-order action until they can.
+   - If Moving, roll 1d6 and: Move away from nearest enemy (1--2), Move towards nearest enemy
+     (3--4), Move towards nearest objective (5--6).
+1. Resolve each activation as normal.
+
+### Arcane Group Activations
+
+Arcane Companies must activate any Groups with the Casting or Summoner order before others, from
+highest to lowest point value. Groups capable of the Casting or Summoning order _must_ use that
+order. If capable of both, alternate between them each time.
+
+### Bold Group Activations
+
+Bold Companies must activate the Group nearest an enemy or objective that has not already activated.
+Groups capable of seizing an objective or attacking an enemy must do so, preferring objectives over
+attacking.
+
+### Cautious Group Activations
+
+Cautious Companies may not Attack an enemy Group whose R is equal to or greater than their own.
+Groups more than 6" away from a friendly Group must Move towards one if possible.
+
+### Ferocious Group Activations
+
+Ferocious Companies must activate the Group nearest an enemy that has not already activated. Groups
+capable of Attacking or Shooting must do so, preferring whichever activation order has a better hit
+chance. Groups not within range of an objective or enemy must Move towards the nearest enemy Group.
+
+### Guarded Group Activations
+
+Guarded Companies activate as normal but may not activate to Attack an enemy Group unless a friendly
+Group is within 12". Groups not within 12" of a friendly Group must move towards the nearest
+friendly Group or defensible position.
+
+## Eccentrics
+
+- Groups in Bold and Ferocious Companies with the Defiant trait must always test to activate it when
+  Attacked.
+- Groups in Cautious and Guarded Companies with the Elusive trait must always test to activate when
+  Attacked.
+- Spells are chosen randomly unless specified beforehand.
+- Groups with the Casting order may not activate to Cast the same spell twice in a row and cast in
+  one of two orders:
+  1. If the Company has the Arcane Personality, cast from highest activation difficulty to lowest if
+     there is a valid target, otherwise cast the next highest difficulty spell. May not recast the
+     same spell until all known spells have been attempted at least once.
+  1. Otherwise, cast randomly against nearest valid target.
+- Groups with Foe or Bane traits must Move towards and attempt to Attack the nearest specified
+  Kind of Group if within 18".
+- Groups with Self-Destruct trait must attempt Self-Destruct once they are Bloodied.
```

Comment:
```suggestion
- Groups in Bold and Ferocious Companies with the Defiant trait must always test to activate that trait when possible.
- Groups in Cautious and Guarded Companies with the Elusive trait must always test to activate when
  possible.
- Choose spells randomly unless specified otherwise.
- Groups with the Casting order may not activate to Cast the same spell twice in a row. If the Company has the Arcane Personality, cast from highest activation difficulty to lowest if
     there is a valid target, otherwise cast the next highest difficulty spell. May not recast the
     same spell until all known spells have been attempted at least once. Otherwise, cast randomly against the nearest valid target.
- Groups with Foe or Bane traits must Move towards and attempt to Attack the nearest specified
  type of Group if within 18" of one.
- Groups with Self-Destruct trait must attempt Self-Destruct once they are Bloodied.
```

Reply:
i really fubared the line breaks on this one, sorry

we need to talk about this stuff with how the Arcane Personality activates. i don't *quite* understand what you're going for. can you ELI5 it to me?

Reply:
Sure, so what I _meant_ to do here is:

1. Keep groups from spamming the same spell over and over
2. Have the arcane groups cycle from their best-to-worst spells where possible

Reply:
okay so i fussed with the arcane personality stuff a bit. its... *a lot* to get in there, and it needs to be super compressed obvs, so tell me 1) if it reads, and 2) if it actually results in what you were going for. i *think* it does, but god knows

Reply:
Scans well enough to me and is as short as I think we can make it without a full rework. Accepting this edit!